src/es_archiver/lib/indices/kibana_index.ts

```diff
@@ -17,29 +17,34 @@
  * under the License.
  */
 
-import _ from 'lodash';
+import { get } from 'lodash';
 import fs from 'fs';
-import path from 'path';
+import Path from 'path';
```

Comment:
TS error `path is already defined in the upper scope` for a lot of (path:string) => something handlers. renaming to `Path` was the easiest solution.